### PR TITLE
[Transaction] Add a check for uninitialized PendingAck

### PR DESCRIPTION
**Pull request by liangyepianzhou**
### Motivation 
We should not generate the statistics of a uninitialized PendingAck,and we should check if it is initialized when we get it by `getStoreManageLedger()`.
### Modifications
 Do not generate the statistics of a uninitialized PendingAck.
 Add check if it is initialized when we get it by `getStoreManageLedger()`.
### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Comment by dave2wave** [comment]
Please update the title to something meaningful

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java [hunk 1]
```diff
@@ -937,6 +937,10 @@ public TransactionInPendingAckStats getTransactionInPendingAckStats(TxnID txnID)
         }
     }
 
+    public boolean checkIfPendingAckStoreInit() {
+        return pendingAckStoreFuture != null;
```
Comment (congbobo184) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java:941]: add check `this.pendingAckStoreFuture.isDone()`

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/TransactionAggregator.java [hunk 2]
```diff
@@ -64,7 +64,8 @@ public static void generate(PulsarService pulsar, SimpleTextOutputStream stream,
                             topic.getSubscriptions().values().forEach(subscription -> {
                                 try {
                                     localManageLedgerStats.get().reset();
-                                    if (!checkTopicIsEventsNames(TopicName.get(subscription.getTopic().getName()))) {
+                                    if (!checkTopicIsEventsNames(TopicName.get(subscription.getTopic().getName()))
+                                            && ((PersistentSubscription) subscription).checkIfPendingAckStoreInit()) {
```
Comment (congbobo184) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/TransactionAggregator.java:68]: we should judge subscription class

**Review by congbobo184 — CHANGES_REQUESTED** [review]

**Review by congbobo184 — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**